db/seeds/development/submissions_seed.rb

```diff
@@ -93,6 +93,7 @@
   guardian_two_phone_type: EnumConstants::PHONE_TYPES.sample,
   guardian_two_relationship: EnumConstants::GUARDIAN_RELATIONSHIPS.sample,
   health_conditions: EnumConstants::HEALTH_CONDITIONS.sample,
+  health_conditions_description: Faker::Name.first_name,
```

Comment:
For this one and others similar, instead of `first_name` seed we can just do `'None'`.